Cargo.toml

```diff
@@ -1,20 +1,31 @@
 [package]
 name = "multitoken"
-version = "0.3.3"
+version = "0.3.4"
 authors = ["Gear Technologies"]
 license = "MIT"
 edition = "2021"
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
-gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.4" }
-gear-lib-derive = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.4" }
+gstd = { git = "https://github.com/gear-tech/gear.git", branch = "testnet" }
+gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.5" }
+gear-lib-derive = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.5" }
 multitoken-io = { path = "io" }
 hashbrown = "0.13"
+gmeta = { git = "https://github.com/gear-tech/gear.git", branch = "testnet" }
 
 [dev-dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", branch = "stable", features = ["debug"] }
-gtest = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
+gstd = { git = "https://github.com/gear-tech/gear.git", branch = "testnet", features = ["debug"] }
+gtest = { git = "https://github.com/gear-tech/gear.git", branch = "testnet" }
+multitoken-state = { path = "state", features = ["binary-vendor"] }
+gclient = { git = "https://github.com/gear-tech/gear.git", branch = "testnet" }
+tokio = "1"
 
 [build-dependencies]
-gear-wasm-builder = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
+gear-wasm-builder = { git = "https://github.com/gear-tech/gear.git", branch = "testnet" }
+multitoken-io = { path = "io" }
+
+[features]
+binary-vendor = []
+
+[workspace]
+members = ["state"]
```

Comment:
I'd ask you to configure your editor to add an empty line at the end of file. E.g. my VS Code settings:

```
    "files.trimTrailingWhitespace": true,
    "files.insertFinalNewline": true,
    "files.trimFinalNewlines": true,
```